### PR TITLE
TRI-56/Add or remove time from a ticket

### PR DESCRIPTION
**Pull request by luccar31**
# Pull Request

## Description
Add endpoints for adding or removing time from a ticket/issue. It can happen if you forgot to pause the timer, for example. Separate endpoint are added for each action.

## Ticket Link
[TRI-56](https://linear.app/tricker-sirius/issue/TRI-56/add-time-remove-time-from-a-ticket)

## Steps to Reproduce
1. Run `prisma generate`
2. Run `docker compose up`
3. Check Swagger documentation for [adding endpoint](http://localhost:8080/api-docs/#/Issue/post_api_issue__issueId__add_time) and [removing endpoint](http://localhost:8080/api-docs/#/Issue/post_api_issue__issueId__remove_time)
4. Test the endpoint with Postman or Swagger.

### CODE REVIEW

File: src/utils/validation-annotations.ts [hunk 1]
```diff
@@ -79,3 +79,33 @@ export function IsValidApiKey(validationOptions?: ValidationOptions) {
     });
   };
 }
+
+/**
+ * Validator constraint class for checking if a date (in form of string) is after or equal to the provided date.
```
Comment (Juan-Bianchi) [inline src/utils/validation-annotations.ts]: Requests are not working, nor with postman neither with Swagger. Please check this validation function. Response obtained is 400 (problems with the date)
image

**Review by Juan-Bianchi — CHANGES_REQUESTED** [review]
Request are not working in neither of both endpoints. Please check validation anotations

File: src/utils/validation-annotations.ts (hunk 1 above)
Comment (luccar31) [reply src/utils/validation-annotations.ts]: Solved.

Comment (Juan-Bianchi) [reply src/utils/validation-annotations.ts]: The same error remains

Comment (luccar31) [reply src/utils/validation-annotations.ts]: Solved x2.

**Review by Juan-Bianchi — APPROVED** [review]

**Pull request merged**